pages/common/xml-elements.md

```diff
@@ -0,0 +1,28 @@
+# xml-elements
+
+> Extract elements and structure of an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Extract elements of an XML document (as XPATH expressions):
+
+`xml elements {{path/to/input.xml|URI}} >{{path/to/elements}}`
+
+- Extract elements and attributes of an XML document:
```

Comment:
Perhaps this will read better?
```suggestion
- Extract elements and attributes from an XML document:
```

Reply:
xml-elements.md:- Extract elements and their attributes from an XML document: